### DISCUSSION TITLE
Need help with contributing

### DISCUSSION THREAD

**Discussion by ImranArif536** [TEAMMATES/teammates#13113] [Help requests] [answered]
Hello,

I have a wish to start contributing to opensource. This project seems to align with my skills, and I want to contribute to the Teammates backend source code. But I am not sure where to start from, because almost all "good-first-issue" issues have a lot of comments and someone has already worked on them.

Can someone please help me push forward in my new journey of open source contributions? I will be so thankful.

**Comment by jayasting98** [comment]
Hi, @ImranArif536. We appreciate your interest in the project.

You can consider having a look at the other issues, even those without the `good-first-issue` label. Some of them may actually be simple enough for beginners, but the team may not have marked them as `good-first-issue` yet.

If you want a suggestion, have a look at #13103. To me, it seems like it should be quite manageable.

**Comment by ImranArif536** [comment]
Hi @jayasting98. Thank you so much for your response and suggestion. It is really helpful. 

Yes, the issue that you mentioned seems to be manageable. Should I get it assigned to myself? And Is there any time constraint in which I should try to do it? Just to let you know, I have not yet set up the project locally. So I will do the local set up and then can pick it up.

Thanks again.

**Comment by jayasting98** [comment, marked as answer]
Feel free to leave a comment on that issue (or any other issue) if you want to try to tackle it @ImranArif536.

>Is there any time constraint in which I should try to do it?

No, not really. However, if you do not work on it for a long time (probably around a few weeks), we may let somebody else do it if anybody else wants to try it.

**Reply by ImranArif536** [reply]
Thanks a lot for the guidance @jayasting98. It was super helpful!
